init.d.optee

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# /etc/init.d/optee
+#
+# Start/stop tee-supplicant (OP-TEE normal world daemon)
+#
+case "$1" in
+    start)
+	if [ -e /bin/tee-supplicant -a -e /dev/teepriv0 ]; then
```

Comment:
Based on (https://github.com/OP-TEE/optee_client/blob/master/tee-supplicant/src/tee_supplicant.c#L388):
```
if (argc == 2) {
		fd = open_dev(argv[1]);
		if (fd < 0) {
			EMSG("failed to open \"%s\"", argv[1]);
			exit(EXIT_FAILURE);
		}
```
we can support different device file names. Maybe it's better to check in the ps list?